### ISSUE TITLE
After mounting some weird file appears

### ISSUE THREAD

**Issue by vtemian** [PressLabs/gitfs#243]
After mount process is over `lextab.py yacctab.py` are present in the directory in which the mounting process started.

**Comment by bogdanpetrea** [comment]
When I run the build command locally (`pex -v --disable-cache -r requirements.txt -e gitfs:mount -o gitfs_output .`) I can see that pycparser contains the lextab and yacctab files.

```
>>> import pycparser
>>> pycparser.__version__
'2.14'
>>> from pycparser import lextab
>>> lextab.__file__
'/home/celo/.pex/install/pycparser-2.14-py2-none-any.whl.c836857f73da51f8e32cdff72cee283d41fee813/pycparser-2.14-py2-none-any.whl/pycparser/lextab.py'
```

From our SFTP(drone build):
```
>>> import pycparser
>>> pycparser.__version__
'2.10'
>>> from pycparser import lextab
Traceback (most recent call last):
  File "<console>", line 1, in <module>
ImportError: cannot import name lextab
>>> pycparser.__file__
'/root/.pex/install/pycparser-2.10-py2-none-any.whl.102d0ccc4a7c33eb95ed7c4d5028509a5662d22c/pycparser-2.10-py2-none-any.whl/pycparser/__init__.pyc'
```

**Comment by vtemian** [comment]
Solved by migrating to pycparser 2.14
